test/cases/020_kernel/003_config_4.11.x/test.sh

```diff
@@ -4,19 +4,20 @@
 # REPEAT:
 
 set -e
+set -x
 
 # Source libraries. Uncomment if needed/defined
 #. "${RT_LIB}"
 . "${RT_PROJECT_ROOT}/_lib/lib.sh"
 
 clean_up() {
-	find . -iname "test-kernel-config*" -not -iname "*.yml" -exec rm -rf {} \;
+    find . -iname "test-kernel-config*" -not -iname "*.yml" -exec rm -rf {} \; || true
```

Comment:
nit: Are we replacing tabs with 4 spaces?
I thought I recall agreement on tabs but I may be mistaken